docs/specification.md

```diff
@@ -35,10 +35,17 @@ base58char := "1" / "2" / "3" / "4" / "5" / "6" / "7" / "8" / "9" / "A" / "B" /
               / "q" / "r" / "s" / "t" / "u" / "v" / "w" / "x" / "y" / "z"
 ```
 
+The permission characters are used to define the power of a did document on other did documents with the same did domain:
+- **A** identifies domain admins that have all the admin power listed below.
+- **C** identifies domain admins that have the power only to create.
+- **U** identifies domain admins that have the power only to update.
+- **D** identifies domain admins that have the power only to deactivate.
+- **T** identifies domain admins that have the power only to make transaction in the scope of the [zenflows project](https://github.com/interfacerproject/zenflows).
```

Comment:
can it be context-free, as in "only make a transaction", up to the domain if that makes sense or discarded?

Reply:
yes, it makes more sense to let it context-free since it can be used also outside of interfacer.